### PR TITLE
Meisterplan Connector Update: New Project Trigger via Webhook Create

### PR DESCRIPTION
**Pull request by itd-sk**
- added endpoint to create webhook for project event trigger
- added endpoint for dynamic notification schema
- added webhook delete endpoint
- added new absences endpoint
- added new resource manager field to resource and role endpoints

---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [ ] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Meisterplan/apiDefinition.swagger.json [hunk 1]
```diff
@@ -88,6 +92,10 @@
...
...A webhook can be registered with a secret. -   When an event occurs, Meisterplan hashes the notification payload     with this secret using HmacSHA256 algorithm and sends the resulting     signature hex encoded with the X-Hook-Signature header. -   The client can verify the authenticity of the notification by     calculating the HmacSHA256 signature and comparing it to the value     of the X-Hook-Signature header. The signature is calculated using     the secret and the payload.\n\nWebhook Callback Acknowledgement\n\nThe client must respond to an event callback with a 200 HTTP status code to acknowledge that the callback was received. The client should respond immediately after receiving the notification. There is a 10 second timeout. If the client responds with an error (except HTTP status code 410), Meisterplan will retry the notification delivery up to 8 times (delayed using exponential backoff). If too many notifications cannot be delivered, the webhook will be suspended. Suspended webhooks are eventually deleted. Meisterplan webhooks do not follow redirects. You can get more information about the notification schema by calling the /v1/meta/notification/dynamicSchema endpoint."
```
Comment (Amjed-Ayoub) [inline certified-connectors/Meisterplan/apiDefinition.swagger.json:98]: @itd-sk Please remove the extra white spaces in the description. please remove from all parameters descriptions in the swagger file

File: certified-connectors/Meisterplan/apiDefinition.swagger.json [hunk 2]
```diff
@@ -1501,7 +1546,396 @@
...
+            "description": "Too Many Requests"
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "Absences"
+        ],
+        "summary": "Create Absences",
+        "description": "Use this request to add an absence for a certain time period. This time period can be defined using the start and finish attributes of the request. Using startDayType or finishDayType, you can specify whether the first/last day of the absence is a full day or only half a day. If the time interval consists of only one day, entering half day in startDayType and/or finishDayType will result in an half day (i.e., if startDayType is half a day and finishDayType is a whole day, then the absence refers to a half day).",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Meisterplan/apiDefinition.swagger.json:1645]: @itd-sk I would suggest you keep all operations' descriptions short to fit in the text box. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]

File: certified-connectors/Meisterplan/apiDefinition.swagger.json (hunk 1 above)
Comment (itd-sk) [reply certified-connectors/Meisterplan/apiDefinition.swagger.json]: @Amjed-Ayoub Removed the white spaces and shortened the description.

File: certified-connectors/Meisterplan/apiDefinition.swagger.json (hunk 2 above)
Comment (itd-sk) [reply certified-connectors/Meisterplan/apiDefinition.swagger.json]: @Amjed-Ayoub shortened description

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @itd-sk,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-updates) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**